priv/repo/migrations/20170729161949_create_repository.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Tudo.Repo.Migrations.CreateRepository do
+  use Ecto.Migration
+
+  def change do
+    create table(:repositories) do
```

Comment:
Yeah, no worries. It won't be typed that many times and we all have auto-completion. 👍